build_package.py

```diff
@@ -338,6 +338,8 @@ def build_win32_installer_package(variant, version):
     make_directory("Scalyr/logs")
     make_directory("Scalyr/data")
     make_directory("Scalyr/config/agent.d")
+    os.chmod("Scalyr/config/agent.d", int("740", 8))
```

Comment:
Same here, let's make a comment so that folks know they shouldn't be mucking with it.